### DISCUSSION TITLE
Drowning in documentation?

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#57] [Look at what I learned!]
Hey guys. I just wanted to throw a couple tips out for those who are feeling a little overwhelmed with all of the documentation and following along with the directions given. I find it very helpful to print the instructions and Read.me files, That way the text is very clear and I have some thing to check off as I'm going along. There is a printer extension in VS code you can install.￼ Below is my example of part of our bio assignment - one is in the readme on vs code, and the other is in the printed format version￼. Same info. You can also print the page on GitHub which works just the same. Hope this helps! 

![image](https://user-images.githubusercontent.com/68397076/106325406-62cd9000-6240-11eb-9942-2c7d7ef3cf7c.jpeg)

**Comment by drteresavasquez** [comment]
Also if you installed the Readme extension that we looked at in class the first week or so, there is an icon in the top right of your code editor with a split screen and magnifying glass. If you click that, you can see the page more clearly as well. All great solutions!

Thanks for sharing this, @danicrosby

**Reply by danicrosby** [reply]
Oh nice!
